auto_backup/models/db_backup.py

```diff
@@ -24,16 +25,18 @@
 class DbBackup(models.Model):
     _name = 'db.backup'
     _description = 'Backup configuration record'
+    _rec_name = 'db_name'
 
     def _get_db_name(self):
-        dbName = self._cr.dbname
-        return dbName
+        db_name = self._cr.dbname
+        return db_name
 
     # Columns for local server configuration
     host = fields.Char('Host', required=True, default='localhost')
     port = fields.Char('Port', required=True, default=8069)
-    name = fields.Char('Database', required=True, help='Database you want to schedule backups for',
+    db_name = fields.Char('Database', required=True, help='Database you want to schedule backups for',
```

Comment:
I'd rather not change field names for compatibility between versions honestly :)

Reply:
@Yenthe666 You're right, I change back the to the previous naming